app/javascript/bundles/Dashboard/components/TaskInfoCard/TasksInfoCard.jsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import PropTypes from "prop-types";
+import _ from "lodash";
+
+import "./tasks_info_card.scss";
```

Comment:
```suggestion
import "./tasks-info-card.scss";
```
for scss files please use ```-``` instead ```_```